prometheus_api_client/metric.py

```diff
@@ -68,9 +69,13 @@ def __init__(self, metric, oldest_data_datetime=None):
             )
             self.metric_values["ds"] = pandas.to_datetime(self.metric_values["ds"], unit="s")
 
-        # Set the metric start time and the metric end time
-        self.start_time = dateparser.parse(str(self.metric_values["ds"].head(1).values[0]))
-        self.end_time = dateparser.parse(str(self.metric_values["ds"].tail(1).values[0]))
+        # Set the metric start time and the metric end time from timestamps in ns
+        self.start_time = datetime.fromtimestamp(
+            self.metric_values["ds"].head(1).values[0].astype(int) * (10 ** -9)
+        )
+        self.end_time = datetime.fromtimestamp(
+            self.metric_values["ds"].tail(1).values[0].astype(int) * (10 ** -9)
```

Comment:
similar comment as above, wouldn't `self.metric_values["ds"].values[-1].astype(int)` do the same?

Reply:
Actually, there is another way that might be better: `self.metric_values.iloc[0,0].astype(int)`

Reply:
This looks better :+1: